buildlib/az-helpers.sh

```diff
@@ -214,15 +214,26 @@ check_release_build() {
     build_reason=$1
     build_sourceversion=$2
     title_mask=$3
+    launch=False
 
-
-    if [ "${build_reason}" == "IndividualCI" ] || \
-       [ "${build_reason}" == "ResourceTrigger" ]
+    # DRP release scheduled testing
+    if [ "${build_reason}" == "Schedule" ] && [[ "$BUILD_DEFINITIONNAME" == *"DRP" ]]
     then
         launch=True
+
+    elif [ "${build_reason}" == "IndividualCI" ] || [ "${build_reason}" == "ResourceTrigger" ]
+    then
+        if [[ "$BUILD_DEFINITIONNAME" == *"DRP" ]]
+        then
+            # Release from DRP only if main pipeline is disabled
+            install_az_tools
```

Comment:
I would call it from check_main_pipeline_status function as it is the one that needs it, in your implementation anyone who calls this function needs to call the install function before which seems redandent and confusing

Reply:
My first take was the same. The output from both functions (install_az_tools, check_main_pipeline_status) got mixed up in Azure, causing the "launch" variable to be assigned a long, meaningless value.

Also, the install function is called only for the DRP pipeline.

Reply:
Do we really have to install tools as part of test pipeline? Can't we do it as part of system setup?

Reply:
Removed `az` tools installation from CI and updated the documentation [(internal link)](https://confluence.nvidia.com/display/NSWX/DRP%3A+UCX+release).